### ISSUE TITLE
Demo page not playing Audio on IOS

### ISSUE THREAD

**Issue by ksnyde** [wle8300/uifx#1]
I'm on an iPad Pro on IOS12 and loaded the site using Chrome browser. The page loaded but the audio did not play on the button press action.

I am on a plane and the network a bit squiffy but page load seemed to be complete before I tried this.

**Comment by wle8300** [comment]
Hi Ken thanks for reporting this bug. I’m seeing this on my end also. This bug seems to affect Chrome/Firefox/Safari iOS.

`uifx` was only tested on desktop browsers.

I don’t have time to fix this rn, but I would welcome a patch if you’re able to fix it.

**Comment by ksnyde** [comment]
If I can find time I will look into but like most of us my time is a rare commodity. Either way, thanks for the contribution. Small effects based sounds are definitely something I feel the modern web could benefit from (although of course overdoing it can be awful 😄 )

**Comment by MrF3lix** [comment]
@wle8300 Thanks a lot for this cool library!

I've experienced similar problems with loading audio files on mobile via the Audio API. One thing that helped me doing this is to call the load function on the HTMLMediaElement.

[https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/load](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/load)

So it would be something like this:
```JavaScript
const audioElement = new Audio(this.asset);
audioElement.load()
```

If you want I could create a PR for this issue.

**Comment by wle8300** [comment]
Oh awesome! Yea! Can you create a PR?

Someone else mentioned this bug last week so I think his bugfix would be a pretty big deal!

**Comment by MrF3lix** [comment]
#2

**Comment by wle8300** [comment]
Thx Felix! Give me a week and I’ll test/merge this. I’ll ping you to let you know!

**Comment by wle8300** [comment]
@ksnyde @MrF3lix This bug has been addressed in `1.0.8`. You can see it on [npm](https://www.npmjs.com/package/uifx).

Thanks @MrF3lix for the PR request!
